### PR TITLE
Add new html files with edge cases, update readme

### PR DESCRIPTION
**Pull request by sp33drac3r**
- Added old version of jQuery to index.html   
- Created two new html files.   
  - One is malformed, missing `<head>` and `</body>` tags.
  - Other names a function `$` in the global scope
- Updated README to emphasize importance of dealing with edge cases.

### CODE REVIEW

File: simplex.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+-->
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Widget Test</title>
+    <script>
+      $ = function() {
+        console.log("You'd have to be a real sociopath to name a funciton $ like this but anythings possible.")
```
Comment (rafaelespinoza) [inline simplex.html]: typos

Comment (rafaelespinoza) [inline simplex.html]: btw, it could also be the wrong version of `jQuery`
